### PR TITLE
Updated Irminger METLGR cal csvs and added NUTNR predeploy cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated METLGR cal csvs to reflect the swapped positions on SUMO buoy:
CGINS-METLGR-00013__2020-08-08 (METBK CT 1 in METBK 2 position)
CGINS-METLGR-00015__2020-08-08 (METBK CT 2 in METBK 1 position)
Also 3 predeploy NUTNR cal csvs:
CGINS-NUTNRB-01091__20200808
CGINS-NUTNRB-01094__20200808
CGINS-NUTNRM-00661__20200808 (for profiling glider 515)

### CODE REVIEW

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20200808.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...75.375, 18587.91666667, 18135.79166667, 17675.58333333, 17242.375, 16817.375, 16413.5, 16028.25, 15681.08333333, 15352.70833333, 15061.83333333, 14802.33333333, 14570.625, 14366.125, 14190.75, 14029.58333333, 13883.75, 13752.41666667, 13610.75, 13471.33333333, 13340.75, 13207.75, 13080.45833333, 12956.375, 12820.58333333, 12707.45833333, 12610.75, 12526.83333333, 12453.45833333, 12403.58333333, 12337.875, 12301.95833333, 12248.625, 12208.70833333, 12187, 12156.79166667, 12124.125, 12097.79166667, 12056.16666667, 12017.70833333, 11961.625, 11899.08333333, 11824.125, 11740.79166667, 11635.91666667, 11521, 11402, 11252, 11092.91666667, 10925.125, 10744.29166667, 10563.20833333, 10377.83333333, 10189.29166667, 9991.625, 9777.29166667, 9526.25, 9271.375, 9002.70833333, 8738.75, 8503.41666667, 8290.95833333, 8089.58333333, 7898.83333333, 7724.58333333, 7563, 7421.83333333, 7288.75, 7162, 7039.125, 6911.375, 6745.16666667, 6575.29166667, 6415.41666667, 6272.79166667, 6141.83333333, 6018.25, 5910.29166667, 5818.125, 5731.79166667, 5655.33333333, 5597.33333333, 5542.45833333, 5485.79166667, 5445.16666667, 5412.20833333, 5384.875, 5332.20833333, 5233.58333333, 5028.41666667, 4624.16666667]",
```
Comment (asmith-whoi) [inline calibration/NUTNRB/CGINS-NUTNRB-01091__20200808.csv:3]: OK

File: calibration/NUTNRM/CGINS-NUTNRM-00661__20200808.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...54.66666667, 26323.29166667, 25939.33333333, 25593.79166667, 25321.20833333, 25069.125, 24852.25, 24666.25, 24514.83333333, 24391.95833333, 24288.45833333, 24197.58333333, 24130.75, 24070.125, 24009.41666667, 23959.95833333, 23930.95833333, 23904.33333333, 23874.625, 23847.25, 23798.25, 23780.66666667, 23770.54166667, 23735.5, 23729.79166667, 23700.625, 23660.08333333, 23599.83333333, 23516.95833333, 23419.95833333, 23322.16666667, 23184.91666667, 23026.83333333, 22868.29166667, 22638.125, 22386.625, 22107.5, 21782.08333333, 21439.04166667, 21059.45833333, 20653.54166667, 20220.08333333, 19770.58333333, 19295.70833333, 18805.79166667, 18328.41666667, 17835.375, 17362.04166667, 16900.58333333, 16453.5, 16015.54166667, 15575.29166667, 15105.33333333, 14621.625, 14128.66666667, 13681.625, 13275.5, 12916.33333333, 12592.25, 12312.33333333, 12063.91666667, 11836.04166667, 11658.08333333, 11486.08333333, 11327.66666667, 11198.25, 11061.29166667, 10890.375, 10682.875, 10473.79166667, 10290.375, 10133.875, 9981.375, 9847.70833333, 9735.70833333, 9632.33333333, 9556.70833333, 9492.75, 9453.58333333, 9419.20833333, 9383.79166667, 9359.625, 9348.08333333, 9238.91666667, 8916.91666667, 8298]",
```
Comment (asmith-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00661__20200808.csv:3]: OK

File: calibration/METBKA/CGINS-METLGR-00013__20200808.csv [hunk 3]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-LGR013,CC_depth_of_conductivity_and_temperature_measurements_m,1.23,GI01SUMO-00007
+LGR013,CC_depth_of_conductivity_and_temperature_measurements_m,1.37,GI01SUMO-00007; METBK 1 CT in METBK 2 position
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00013__20200808.csv:2]: Looks good, changed the comma to a semicolon in the notes column.

File: calibration/METBKA/CGINS-METLGR-00015__20200808.csv [hunk 4]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-LGR015,CC_depth_of_conductivity_and_temperature_measurements_m,1.37,GI01SUMO-00007
+LGR015,CC_depth_of_conductivity_and_temperature_measurements_m,1.23,GI01SUMO-00007; METBK 2 CT in METBK 1 position
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00015__20200808.csv:2]: Looks good, changed the comma to a semicolon in the notes column.

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20200808.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
...9, 272.82, 273.64, 274.46, 275.28, 276.1, 276.93, 277.75, 278.57, 279.39, 280.21, 281.04, 281.86, 282.68, 283.5, 284.32, 285.15, 285.97, 286.79, 287.61, 288.43, 289.26, 290.08, 290.9, 291.72, 292.54, 293.37, 294.19, 295.01, 295.83, 296.65, 297.47, 298.3, 299.12, 299.94, 300.76, 301.58, 302.4, 303.22, 304.05, 304.87, 305.69, 306.51, 307.33, 308.15, 308.97, 309.79, 310.61, 311.43, 312.26, 313.08, 313.9, 314.72, 315.54, 316.36, 317.18, 318, 318.82, 319.64, 320.46, 321.28, 322.1, 322.92, 323.74, 324.55, 325.37, 326.19, 327.01, 327.83, 328.65, 329.47, 330.29, 331.1, 331.92, 332.74, 333.56, 334.38, 335.19, 336.01, 336.83, 337.65, 338.46, 339.28, 340.1, 340.92, 341.73, 342.55, 343.37, 344.18, 345, 345.81, 346.63, 347.44, 348.26, 349.08, 349.89, 350.71, 351.52, 352.34, 353.15, 353.96, 354.78, 355.59, 356.41, 357.22, 358.03, 358.85, 359.66, 360.47, 361.28, 362.1, 362.91, 363.72, 364.53, 365.34, 366.16, 366.97, 367.78, 368.59, 369.4, 370.21, 371.02, 371.83, 372.64, 373.45, 374.26, 375.07, 375.88, 376.69, 377.49, 378.3, 379.11, 379.92, 380.73, 381.53, 382.34, 383.15, 383.95, 384.76, 385.57, 386.37, 387.18, 387.98, 388.79, 389.59, 390.4, 391.2, 392.01, 392.81, 393.61, 394.42, 395.22, 396.02]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01091__20200808.csv:8]: Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00661__20200808.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
... 271.26, 272.07, 272.88, 273.69, 274.5, 275.32, 276.13, 276.94, 277.75, 278.56, 279.38, 280.19, 281, 281.81, 282.62, 283.44, 284.25, 285.06, 285.87, 286.68, 287.5, 288.31, 289.12, 289.93, 290.75, 291.56, 292.37, 293.18, 293.99, 294.81, 295.62, 296.43, 297.24, 298.05, 298.87, 299.68, 300.49, 301.3, 302.11, 302.92, 303.74, 304.55, 305.36, 306.17, 306.98, 307.79, 308.61, 309.42, 310.23, 311.04, 311.85, 312.66, 313.47, 314.28, 315.1, 315.91, 316.72, 317.53, 318.34, 319.15, 319.96, 320.77, 321.58, 322.39, 323.2, 324.01, 324.82, 325.63, 326.44, 327.25, 328.06, 328.87, 329.68, 330.49, 331.3, 332.11, 332.92, 333.73, 334.54, 335.34, 336.15, 336.96, 337.77, 338.58, 339.39, 340.19, 341, 341.81, 342.62, 343.43, 344.23, 345.04, 345.85, 346.65, 347.46, 348.27, 349.07, 349.88, 350.69, 351.49, 352.3, 353.1, 353.91, 354.72, 355.52, 356.33, 357.13, 357.94, 358.74, 359.55, 360.35, 361.15, 361.96, 362.76, 363.57, 364.37, 365.17, 365.98, 366.78, 367.58, 368.38, 369.19, 369.99, 370.79, 371.59, 372.39, 373.2, 374, 374.8, 375.6, 376.4, 377.2, 378, 378.8, 379.6, 380.4, 381.2, 382, 382.8, 383.6, 384.39, 385.19, 385.99, 386.79, 387.59, 388.38, 389.18, 389.98, 390.77, 391.57, 392.37, 393.16, 393.96, 394.75]",
```
Comment (snwhite) [inline calibration/NUTNRM/CGINS-NUTNRM-00661__20200808.csv:8]: Looks good.

**Pull request merged**